OKTA-549236: Updating dev docs to reflect hiding sensitive data in Splunk token field (#3746)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md

    "status": "ACTIVE",
    "settings": {

>       "host": "acme.splunkcloud.com",

Reviewer: ```suggestion
      "host": "acme.splunkcloud.com"
```

```json
{
  "host": "acme.splunkcloud.com",

>   "token": "11111111-1111-2222-2222-222222222222"

Reviewer: ```suggestion
  "token": "11111-redacted"
```
Is the format of the returned token `#####-redacted` or just hashed-out, like `********` ?

Author: Upon discussion on the API shape, the token field will not be returned from the API at all

Reviewer: Ok, we can leave it as-is for a creation example.

| Property      | Description                                                  | DataType                                                       | Nullable | Unique | Readonly | MinLength | MaxLength |
| ------------- | ------------------------------------------------------------ | -------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- |
| host            | The domain name for your Splunk Cloud instance. Don't include `http` or `https` in the string. For example: `acme.splunkcloud.com`                                       | String                                                         | FALSE    | FALSE   | FALSE     |      17     |     116      |

> | token     | The HEC token for your Splunk Cloud HTTP Event Collector. The token value is not returned from Okta to protect sensitive data.            | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |

Reviewer: ```suggestion
| token     | The HEC token for your Splunk Cloud HTTP Event Collector. The token value is set during object creation, but isn't returned.            | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```

| Property      | Description                                                  | DataType                                                       | Nullable | Unique | Readonly | MinLength | MaxLength |
| ------------- | ------------------------------------------------------------ | -------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- |
| host            | The domain name for your Splunk Cloud instance. Don't include `http` or `https` in the string. For example: `acme.splunkcloud.com`                                       | String                                                         | FALSE    | FALSE   | FALSE     |      17     |     116      |

> | token     | The HEC token for your Splunk Cloud HTTP Event Collector. Token value returned from Okta will be redacted to protect the sensitive data            | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |

Reviewer: ```suggestion
| token     | The HEC token for your Splunk Cloud HTTP Event Collector. The token value returned from Okta is redacted to protect sensitive data.           | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```
